scripts/install_protolint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Define the Protolint version and file name
+PROTO_VERSION="0.47.4"
+FILE_NAME="protolint_${PROTO_VERSION}_linux_amd64.tar.gz"
+
+# Download Protolint and checksums
+curl -sSOL "https://github.com/yoheimuta/protolint/releases/download/v${PROTO_VERSION}/${FILE_NAME}"
```

Comment:
It looks like this downloads the file to the current working directory. Should we put it into `/tmp` instead?

Reply:
Yes, that would be a more clean way. Thanks!